development.md

```diff
@@ -81,9 +81,9 @@ Our repository fingerprint is:
 
 ## Translation of c:geo
 
-If you want to contribute translations, you need a free account at [Crowdin](https://crowdin.com/). Then start contributing new translations to or vote for existing translation proposals in the [c:geo translation project](http://translate.cgeo.org). If you would like to include a new language please contact us via Crowdin.
+If you want to contribute translations, you need a free account at [Crowdin](https://crowdin.com/). Then start contributing new translations to or vote for existing translation proposals in the [c:geo translation project](https://translate.cgeo.org). If you would like to include a new language please contact us via Crowdin.
```

Comment:
It seems translate.cgeo.org is not working with https.
The CNAME is configured on Crowdin, not sure whats wrong. Anyway we could just link to Crowdins c:geo project page from there.

Reply:
The problem is here:
![grafik](https://github.com/cgeo/cgeo.github.io/assets/32555742/1fc40111-7dd6-4785-928d-c8e968fc73f7)

I will change the link to crowdin.

Reply:
> I will change the link to crowdin.

Done.